src/main/java/com/puppycrawl/tools/checkstyle/xpath/XpathQueryGenerator.java

```diff
@@ -135,15 +169,19 @@ private static String generateXpathQuery(DetailAST ast) {
      * @param root {@code DetailAST} root ast
      * @param lineNumber line number
      * @param columnNumber column number
+     * @param fileText {@code FileText} object
+     * @param tabWidth distance between tab stop position
      * @return list of nodes matching defined line and column number
      */
     private static List<DetailAST> getMatchingAstElements(DetailAST root, int lineNumber,
-                                                          int columnNumber) {
+                                                          int columnNumber, FileText fileText,
```

Comment:
@MEZk @timurt Shouldn't we just remove the `static` and use the class fields directly?
Why do we drop the instance and pass all the class fields as parameters like this?

Reply:
@rnveach 
Yes, it will be better. @timurt, please do for this method.

Reply:
done